rimage/lazy_encoded.go

```diff
@@ -3,12 +3,15 @@ package rimage
 import (
 	"bytes"
 	"context"
+	"fmt"
 	"image"
 	"image/color"
 	"net/http"
 	"strings"
 	"sync"
 
+	"go.uber.org/multierr"
```

Comment:
I recommend using errors.Join from the standard library rather than multierr

Reply:
Done